src/commands/fun/rollDice.ts

```diff
@@ -0,0 +1,45 @@
+import { Message } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+
+class RollDiceCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'roll-dice',
+      aliases: ['rd', 'roll', 'rolldice', 'dice-roll', 'diceroll', 'dice'],
+      group: 'fun',
+      args: [
+        {
+          key: 'sides',
+          prompt: 'enter the number of sides of the dice',
+          type: 'integer'
+        }
+      ],
+      memberName: 'dice-coin',
+      description: 'Roll a dice!',
+      examples: [`${client.commandPrefix}dice-roll`]
+    });
+  }
+
+  getRandomInt(max: number): number {
```

Comment:
I think this gives you [1, max) instead of [1, max]

Reply:
yes this is a problem

Reply:
it should be fixed now

Reply:
This is still not quite right, the probability of rolling a 1 is higher than all other options

Reply:
This is very bad